examples/automm/tabular_dl/example_tabular.py

```diff
@@ -35,8 +35,10 @@
 
 automm_hyperparameters = {
     "data.categorical.convert_to_text": False,
-    "model.names": ["categorical_transformer", "numerical_transformer", "fusion_transformer"],
-    "model.numerical_transformer.embedding_arch": ["linear"],
+    # "model.names": ["categorical_transformer", "numerical_transformer", "fusion_transformer"],
+    # "model.numerical_transformer.embedding_arch": ["linear"],
+    "model.names": ["ft_transformer_refactor"],
```

Comment:
`ft_transformer_refactor ` - > `ft_transformer`. Change other places accordingly.

Reply:
Need to remove commented code.